src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,80 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driver1 = driverService.create(new Driver("Bob", "1234ew22q"));
+        Driver driver2 = driverService.create(new Driver("John", "44eet34"));
+        Driver driver3 = driverService.create(new Driver("Mark", "24fet37"));
+
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        List<Driver> driversBmw = new ArrayList<>();
+        driversBmw.add(driver1);
+        driversBmw.add(driver2);
+        driversBmw.add(driver3);
+
+        Manufacturer bmw = manufacturerService.create(new Manufacturer("BMW", "Germany"));
+        Car bmwX5Car = new Car();
+        bmwX5Car.setModel("X5");
+        bmwX5Car.setManufacturer(bmw);
+        bmwX5Car.setDrivers(driversBmw);
+
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        bmwX5Car = carService.create(bmwX5Car);
```

Comment:
car object is mutable
so we don't need to reset the value - we will be able to get id anyway
```suggestion
        carService.create(bmwX5Car);
```

Reply:
а если нам id база другой даст?